pom.xml

```diff
@@ -560,24 +562,24 @@ http://www.w3.org/2001/XMLSchema-instance">
                 <artifactId>maven-changelog-plugin</artifactId>
                 <version>2.3</version>
             </plugin>
-<!--
-            <plugin>
-                <groupId>com.github.jinnovations</groupId>
-                <artifactId>attribution-maven-plugin</artifactId>
-                <version>0.9.4</version>
-                <executions>
-                    <execution>
-                        <goals>
-                            <goal>generate-attribution-file</goal>
-                        </goals>
-                        <phase>generate-resources</phase>
-                        <configuration>
-                            <outputFile>${user.dir}/ATTRIBUTIONS/${project.artifactId}-attribution.xml</outputFile>
-                        </configuration>
-                    </execution>
-                </executions>
-            </plugin>
--->
+            <!--
+                        <plugin>
+                            <groupId>com.github.jinnovations</groupId>
+                            <artifactId>attribution-maven-plugin</artifactId>
+                            <version>0.9.4</version>
+                            <executions>
+                                <execution>
+                                    <goals>
+                                        <goal>generate-attribution-file</goal>
```

Comment:
@gittify I don't know why has this been preserved. This is reflecting here because of a formatting change. @jcwuzoegiver or @iizrailevsky might be able to confirm.